### ISSUE TITLE
[Accessibility] A11y_ Java Tooling for IntelliJ/Eclipse_Eclipse Azure Sign In (Dark Theme): In Dark theme, text under the unselected radio button is blurred

### ISSUE THREAD

**Issue by Tanushagottipati** [microsoft/azure-tools-for-java#3313]
[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Faccessibilityinsights.io%2F&data=02%7C01%7Cv-tagott%40microsoft.com%7Cb67b2c4b646d4f9561a208d6f4b5c39b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636965458847261333&sdata=PJMiKjb64uwhBN98LZpctnBiAVeOvw%2BLBrnxBtrjl2M%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier

GitHubTags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_Java Tooling for IntelliJ/Eclipse-Stay Healthy; #MAS1.4.3; #Color Contrast; #C+E-H; #Java tooling for IntelliJ/Eclipse

**Environment Details:**
Eclipse IDE for Enterprise Java Developers.
Version: 2019-06 (4.12.0)
Build id: 20190614-1200
Solution Installed: The Azure Toolkit for Eclipse 3.17.0

**OS Details:** 
OS: Windows (Version 1903, Build: 18362.175)
MAS Violated: MAS1.3.1

**Note:** This issue is found for the text that is below the unselected radio button
            Similar Issue is in **‘New Dynamic Web App’** popup, **“Deploy a Webapp”** popup, **"Create New Web App"** popup

**Repro Steps:** 
1) Open Java EE - Eclipse application and change the theme to dark in (WindowsPreferencesGeneralappearancesSelect Dark theme)
2) Install 'Azure Toolkit for Eclipse' from Marketplace
3) Tab till tools-->Azure-->Azure Sign in and press enter
4) Verify if text under the unselected radio button is blurred

**Actual Result:** 
In Dark theme, text under the unselected radio button is blurred

**Expected Result:** 
In Dark Theme all the Text present in the popup must be visible clearly to the user to read it properly

**User Impact:** 
Users will not be able to get the information properly if the text is blurred and thereby will not be able to select the radio button

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

![Bug_2_Deploy a webapp](https://user-images.githubusercontent.com/41109135/62202974-ed150980-b3a7-11e9-82d6-816eba8bc087.png)
![Bug_2_New Dynamic Web App](https://user-images.githubusercontent.com/41109135/62202975-ed150980-b3a7-11e9-9368-819a7bf2bb9f.png)
![Bug_2_text is blurred in dark theme (2)](https://user-images.githubusercontent.com/41109135/62202976-edada000-b3a7-11e9-8e4a-67bba2167422.png)
![Bug_2_text is blurred in dark theme](https://user-images.githubusercontent.com/41109135/62202977-edada000-b3a7-11e9-9513-2012d368ac1e.png)

**Comment by Flanker32** [comment]
The appearance of disabled label is controlled by Eclipse theme, which affect all ui in Eclipse and is out of the scope of Azure Toolkits. Besides, Eclipse provide multiple theme and this issue will not happen in other theme, for example, Light, here are the screen shots.
*Sign in:*
![image](https://user-images.githubusercontent.com/12445236/63920346-36be4600-ca73-11e9-9d04-035e1c051f7d.png)

*Create dynamic web project:*
![image](https://user-images.githubusercontent.com/12445236/63920383-4e95ca00-ca73-11e9-8885-766513235059.png)

*Deploy WebApp:*
![image](https://user-images.githubusercontent.com/12445236/63920477-9157a200-ca73-11e9-934c-d197fb0e428e.png)

**Comment by Flanker32** [comment]
- Importance of experience: Medium,  the text is blurred which may make users not be able to get the information and make selection.
- Quality of experience: not impacted. Eclipse provide alter theme for users, for instance, light, which has more clear fonts for disabled jabels.
- Alternate solutions: No alternate technologies or options can be leveraged.
- Evidence: All evidence is provided in this issue and external issue.

**Comment by Flanker32** [comment]
@Tanushagottipati  Please have a review,

**Comment by soumyatippapuram** [comment]
@Flanker32 Please provide the link to your AAT. This is required to follow Exclusion process. For more details, please refer : https://microsoft.sharepoint.com/teams/ceacc/SitePages/External%20Bug%20Process.aspx

**Comment by Flanker32** [comment]
Tracking in AAT [here](https://cela-ra-aat-prod.azurewebsites.net/#/workflow/56452990-0d38-45bc-b475-a62832ce0076), @soumyatippapuram  please help us review it.

**Comment by Tanushagottipati** [comment]
External Bug: https://bugs.eclipse.org/bugs/show_bug.cgi?id=550842

**Comment by Flanker32** [comment]
Discussed this bug with Ryan and Sarah and approved to close this issue as its external and there is non blocking scenario.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749739
